Sources/StreamChat/Database/DTOs/MessageDTO_Tests.swift

```diff
@@ -627,6 +627,61 @@ class MessageDTO_Tests: XCTestCase {
         XCTAssert(result is ClientError.ChannelDoesNotExist)
     }
     
+    func test_creatingNewMessage_shouldUpdateChannelDatesToTriggerSorting() throws {
+        // Prepare the current user and channel first
+        let cid: ChannelId = .unique
+        let currentUserId: UserId = .unique
+        
+        _ = try await { completion in
+            database.write({ (session) in
+                try session.saveCurrentUser(payload: .dummy(
+                    userId: currentUserId,
+                    role: .admin,
+                    extraData: NoExtraData.defaultValue
+                ))
+                
+                try session.saveChannel(payload: self.dummyPayload(with: cid))
+                
+            }, completion: completion)
+        }
+        
+        // Create a new message
+        var newMessageId: MessageId!
+        
+        let newMessageText: String = .unique
+        let newMessageCommand: String = .unique
+        let newMessageArguments: String = .unique
+        let newMessageParentMessageId: String = .unique
+        let newMessageAttachmentSeeds: [_ChatMessageAttachment<NoExtraDataTypes>.Seed] = [
+            .dummy(),
+            .dummy()
+        ]
```

Comment:
Why bother we these parameters when they are optional and they are not part of the test?